rnd/autogpt_server/autogpt_server/blocks/agent.py

```diff
@@ -107,6 +107,8 @@ class Output(BlockSchema):
     def __init__(self):
         super().__init__(
             id="d2e2ecd2-9ae6-422d-8dfe-ceca500ce6a6",
+            description="AutoGPT agent, it utilizes a Large Language Model and enabled components/tools to perform a task.",
+            categories={BlockCategory.LLM},
```

Comment:
Is this the right category? AutoGPT can do more than LLMs right?

Reply:
LLM just groups all blocks that utilize llm. 

Currently, there is no restriction on a single block to be part of only 1 category.